### PR TITLE
Add browser check back to landing page

### PR DESCRIPTION
**Pull request by jonescc**

### CODE REVIEW

**Comment by julian1** [comment]
Are we sure we don't want to make an exception for MS Edge?

**Comment by julian1** [comment]
It seems to work pretty well.

**Comment by jonescc** [comment]
@pblain?

File: web-app/js/portal/utils/BrowserCheck.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+function browserCheck() {
+	var supported = false; // exclusive
```
Comment (julian1) [inline web-app/js/portal/utils/BrowserCheck.js]: would it make sense to move this default var assigment to the place where it's actually being used - in the (isChrome || isDirefox) tlogic?

Comment (julian1) [inline web-app/js/portal/utils/BrowserCheck.js]: i don't really mind either way

File: web-app/js/portal/utils/BrowserCheck.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+	var isChrome = !!window.chrome;
```
Comment (julian1) [inline web-app/js/portal/utils/BrowserCheck.js]: what is the double negative for?

**Comment by pblain** [comment]
Let's not make an exception for MS Edge because it's not officially supported (event though it works).

**Pull request merged**